packages/store-sdk/src/search/serializer.ts

```diff
@@ -25,10 +25,12 @@ export const format = (params: SearchParamsState): URL => {
     query.push('per')
   }
 
-  const url = new URL(
-    `${params.base}${query.join('/')}`,
-    window.location.origin
-  )
+  if (typeof params.page === 'number') {
+    map.push('page')
+    query.push(params.page.toString())
+  }
+
+  const url = new URL(`${params.base}${query.join('/')}`, 'http://localhost')
```

Comment:
Should this be localhost?

Reply:
I had to do this so it works on SSR. Since we don't really use the host anywhere this shouldn't really be a problem